### ISSUE TITLE
TypeError: mime.extension is not a function

### ISSUE THREAD

**Issue by Pranav-Dakshina** [chirag04/mail-listener2#75]
I get this error when I run Mail-listener2, I guess this is because mailparser in the dependencies is ~0.4.6, The latest mailparser solves this issue.

**Comment by andreihaiducu** [comment]
I could easily fix this with mime.getExtension(contentType)

**Comment by Pranav-Dakshina** [comment]
@andreihaiducu Yeah that works too. I think they have added this change in mailparser 0.6.2.

**Comment by andreihaiducu** [comment]
can you tell me which version of mail-listener i can get ? I'm trying to implement a mail listener but can't find a good version.
Tried mail-listener3 and mail-listener2 both give same error : mime.extension not a function .
Thanks

**Comment by Pranav-Dakshina** [comment]
@andreihaiducu try mail-listener4. I have added this change in that.

**Comment by VivekPowar** [comment]
@Pranav-Dakshina how to download attachment file in attachment.path is undefined
mailListener.on("attachment", function(attachment){
  console.log(attachment.path);
});
